.github/workflows/check-D-labels.yml

```diff
@@ -0,0 +1,51 @@
+name: Check D labels
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, unlabeled]
+    paths:
+      - runtime/polkadot/**
+      - runtime/kusama/**
+      - runtime/common/**
+      - primitives/src/**
+
+jobs:
+  check-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull image
+        env:
+          IMAGE: paritytech/ruled_labels:0.3.2
+        run: docker pull $IMAGE
+
+      - name: Check labels
+        env:
+          IMAGE: paritytech/ruled_labels:0.3.2
+          MOUNT: /work
+          GITHUB_PR: ${{ github.event.pull_request.number }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          API_BASE: https://api.github.com/repos
+          REPO: ${{ github.repository }}
+          RULES_PATH: labels/ruled_labels
+          CHECK_SPECS: specs_polkadot.yaml
+        run: |
+          echo "REPO: ${REPO}"
+          echo "GITHUB_PR: ${GITHUB_PR}"
+          # Clone repo with labels specs
+          git clone https://github.com/paritytech/labels
+          # Fetch the labels for the PR under test
+          labels=$( curl -H "Authorization: token ${GITHUB_TOKEN}" -s "$API_BASE/${REPO}/pulls/${GITHUB_PR}" | jq '.labels | .[] | .name' | tr "\n" ",")
+
+          if [ -z "${labels}" ]; then
+            echo "No labels found. Please add labels D labels"
+            exit 1
+          fi
```

Comment:
You may do without this. Starting with `v0.3.2`, `ruled_labels` has a new flag: `--no-label`. You can run a check eirther passing `--labels` or `--no-label`.

Doing so maybe slightly better as you leave the decision to the rules. 
For instance, if we change the rules and the current `D` labels are renamed into `K`, you would not need to modify this script.

Reply:
Fixed